agent/pkg/api/socket_routes.go

```diff
@@ -44,6 +44,7 @@ func init() {
 	connectedWebsockets = make(map[int]*SocketConnection, 0)
 }
 
+// TODO: add m2m authentication to these routes
```

Comment:
do we have a jira ticket not to forget to do it?
In "standalone mode" - we should secure the websockets and all api server endpoints

Reply:
Created, TRA-4134